apps/admin/src/app/login/page.tsx

```diff
@@ -12,6 +12,12 @@ const LoginPage = () => {
   const { loginAdminData, handleLoginAdminDataChange } = useInput();
   const { handleLoginButtonClick } = useLoginAction(loginAdminData);
 
+  const EnterKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  const handleEnterKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
```

모든 핸들러 함수에 앞에 handle을 붙이는게 마루의 컨벤션이요 일관성이 맞으면 전체적으로 코드가 깔끔해지고 다른 개발자가 보았을때 헷갈리는 면도 줄어들 것 같아요!